src/data-import.ts

```diff
@@ -0,0 +1,331 @@
+/**
+ * DataImportService
+ * Handles inventory data import functionality for cross-device synchronization
+ */
+
+import type { InventoryItems } from './inventory.js';
+import type { ExportData } from './data-export.js';
+
+export interface ImportData extends ExportData {
+  // Import data has the same structure as export data
+}
+
+export interface ImportOptions {
+  mergeStrategy: MergeStrategy;
+  validateSchema: boolean;
+  createBackup: boolean;
+}
+
+export enum MergeStrategy {
+  REPLACE = 'replace', // Replace entire inventory
+  MERGE = 'merge', // Merge quantities (add to existing)
+  SKIP_EXISTING = 'skip_existing', // Only import new colors
+}
```

Comment:
I'm not interested in handling different merge strategies for now.
Remove this type, and rewrite the code to always replace the existing data with the new one. Make sure that the new implementation is as simple as possible.